cmake/modules/hunter_get_toolchain_binaries.cmake

```diff
@@ -48,7 +48,10 @@ function(hunter_get_toolchain_binaries)
         CMAKE_C_PREPROCESSOR
         "${CMAKE_C_ANDROID_TOOLCHAIN_PREFIX}cpp${CMAKE_C_ANDROID_TOOLCHAIN_SUFFIX}"
     )
-    if(NOT EXISTS "${CMAKE_C_PREPROCESSOR}")
+    # GCC has been removed from NDK r18.
+    if(${ANDROID_NDK_MAJOR} VERSION_GREATER 17)
```

Comment:
This variable is not defined for CMake based Android toolchains and I see no analogous variable there. If you don't have any better ideas I suggest just to check `if(NOT EXISTS "${CMAKE_C_PREPROCESSOR}")`.